docker/helm/values.yaml

```diff
@@ -28,6 +28,18 @@ image:
   # Overrides the image tag whose default is the chart appVersion.
   tag: "master-snapshot"
 
+probe:
+  liveness:
+    enabled: true
+    failureThreshold: 10
+    initialDelaySeconds: 30
+    periodSeconds: 10
+  readiness:
+    enabled: true
+    failureThreshold: 10
+    initialDelaySeconds: 30
+    periodSeconds: 10
```

Comment:
can we keep the same ordering with `kyuubi-depolyment.yaml` ?
```
initialDelaySeconds
periodSeconds
failureThreshold
```

Reply:
@ulysses-you Done, thanks.